### PR TITLE
Portal v2 onboarding polish 

### PR DESCRIPTION
**Pull request by fragosti**
## Description

Ideally, this is the last branch before we consider onboarding ready-to-ship (at least v1).
- Tooltip always points to wallet, and so jumps around less. 
- Fixed issue where mobile onboarding did not animate correctly on Webkit based browsers (Safari + any iOS browser).
- Changed image for final onboarding step.
- Can now click outside to close onboarding. 
- Broke up some steps so that they can fit on mobile (specifically wrap ETH).
- Fixed issue where we don't scroll to wallet on mobile onboarding if onboarding starts automatically.
- Made some steps a bit more customized depending on what you've done already.
- Fixed an issue where the `skip` button was on the wrong side of the onboarding card.
- More z-index issues!
- Always start onboarding from 0.
- Go directly to relayer page on mobile after you're done with onboarding.
- Always automatically show onboarding if you're on the 0th (and therefore don't have metamask installed) step.

We decided not to skip steps for now, since ideally the majority of people going through the flow will be starting from scratch, and it' always nice to go back through the steps if you've done them already.

screen shot 2018-06-27 at 6 28 08 pm

## Testing instructions
Try it out and see if the changes above work. It has been deployed to dogfood and feels pretty good on Cipher.

## Types of changes

*   [ ] Bug fix (non-breaking change which fixes an issue)
*   [x] New feature (non-breaking change which adds functionality)
*   [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

*   [ ] Change requires a change to the documentation.
*   [ ] Added tests to cover my changes.
*   [ ] Added new entries to the relevant CHANGELOG.jsons.
*   [ ] Labeled this PR with the 'WIP' label if it is a work in progress.
*   [x] Labeled this PR with the labels corresponding to the changed package.

### CODE REVIEW

**Comment by fragosti** [comment]
@BMillman19 all feedback should be addressed except for detecting metamask when it is installed. Comment from quip:
```
This seems to be pretty hard. I tried polling for an injected web3, but metamask only injects that on page load. It might be possible to use some API to check whether a certain extension is installed, but it certainly doesn't seem to be a standard thing after some googling - also we would have to implement this for each browser.
What might be preferable is: for the special case of step === 0, we always automatically show onboarding (or maybe show it .3 times?), such that a user could visit, install metamask, come back and refresh and start at the correct position in onboarding.
```

File: packages/website/ts/components/onboarding/add_eth_onboarding_step.tsx [hunk 1]
```diff
@@ -1,18 +1,41 @@
...
+            </Text>
+            <Container marginTop="15px" marginBottom="15px">
+                <img src="/images/ether_alt.svg" height="50px" width="50px" />
+            </Container>
+        </div>
+    ) : (
+        <div className="flex items-center flex-column">
+            <Text> Before you begin you will need to send some ETH to your wallet.</Text>
+            <Container marginTop="15px" marginBottom="15px">
+                <img src="/images/ether_alt.svg" height="50px" width="50px" />
```
Comment (BMillman19) [inline packages/website/ts/components/onboarding/add_eth_onboarding_step.tsx]: Should we use our `Image` component here?

File: packages/website/ts/components/onboarding/add_eth_onboarding_step.tsx [hunk 2]
```diff
@@ -1,18 +1,41 @@
...
+            </Container>
+            <Text className="xs-hide">
+                Click on the <img src="/images/metamask_icon.png" height="20px" width="20px" /> metamask extension in
```
Comment (BMillman19) [inline packages/website/ts/components/onboarding/add_eth_onboarding_step.tsx]: "MetaMask"

Comment (BMillman19) [inline packages/website/ts/components/onboarding/add_eth_onboarding_step.tsx]: `Image` component here too

File: packages/website/ts/components/onboarding/intro_onboarding_step.tsx [hunk 3]
```diff
@@ -7,9 +7,13 @@ export interface IntroOnboardingStepProps {}
 export const IntroOnboardingStep: React.StatelessComponent<IntroOnboardingStepProps> = () => (
     <div className="flex items-center flex-column">
         <Text>
-            In order to start trading on any 0x relayer in the 0x ecosystem, you need to complete two simple steps.
+            In order to start trading on any 0x relayer in the 0x ecosystem, you need to complete three simple steps.
         </Text>
         <Container width="100%" marginTop="25px" marginBottom="15px" className="flex justify-around">
+            <div className="flex flex-column items-center">
+                <img src="/images/ether.png" height="50px" width="50px" />
```
Comment (BMillman19) [inline packages/website/ts/components/onboarding/intro_onboarding_step.tsx]: Same comments about Image

File: packages/website/ts/components/wallet/wallet.tsx [hunk 4]
```diff
@@ -325,14 +311,14 @@ export class Wallet extends React.Component<WalletProps, WalletState> {
...
-        return this._renderBalanceRow(key, icon, primaryText, secondaryText, accessoryItemConfig, false, 'eth-row');
+        return this._renderBalanceRow(key, icon, primaryText, secondaryText, accessoryItemConfig, false);
     }
     private _renderTokenRows(): React.ReactNode {
         const trackedTokens = this.props.trackedTokens;
         const trackedTokensStartingWithEtherToken = trackedTokens.sort(
             firstBy((t: Token) => t.symbol !== constants.ETHER_TOKEN_SYMBOL)
                 .thenBy((t: Token) => t.symbol !== constants.ZRX_TOKEN_SYMBOL)
-                .thenBy('trackedTimestamp'),
+                .thenBy('address'),
```
Comment (BMillman19) [inline packages/website/ts/components/wallet/wallet.tsx]: How come we changed this back? May have been a mistake when merging

File: packages/website/ts/components/wallet/wallet.tsx [hunk 5]
```diff
@@ -397,7 +375,7 @@ export class Wallet extends React.Component<WalletProps, WalletState> {
         const style = { ...styles.tokenItem, ...additionalStyle };
         const etherToken = this._getEthToken();
         return (
-            <div id={key} key={key} className={`flex flex-column ${className || ''}`}>
+            <div key={key} className={`flex flex-column ${className || ''}`}>
```
Comment (BMillman19) [inline packages/website/ts/components/wallet/wallet.tsx]: Same here

File: packages/website/ts/components/wallet/wallet.tsx (hunk 4 above)
Comment (fragosti) [reply packages/website/ts/components/wallet/wallet.tsx]: Oops. This was git.

**Pull request merged**